.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx --no lint-staged
+npx --no-install lint-staged
```

Comment:
Maybe better leave it with `--no` and 
1. Add to the instructions in the README that you need npm 7+ _or_
2. Add a check here for the `npx`-Version and `echo` a message to the user, if the version is to old (flexing your shell skills)

Reply:
I was curious, this should work for `2.`:

```suggestion
version=$(npm -v) \
&& [[ $version[1] = 7 ]] \
|| ( echo '\n\n ERROR: You need at least npm version 7 to work with latest husky\n\n'; exit 1 )

npx --no lint-staged
```

Reply:
I only saw your remarks now. I did a bit of research and thought this might also be done with a `.npmrc` and `engines` defined in the `package.lock`. I tested this with an older npm version and it worked. How do you think about that?